lib/index.js

```diff
@@ -331,22 +341,31 @@ function _validateReceiveHeader(header, value) {
       header + ' was not in the request', 'SyntaxError');
   }
 }
+function addValue({result, validate, key, values, label}) {
+  let value = values[key];
+  validate(key, value);
+  if(Array.isArray(value)) {
+    value = value.join(', ');
+  }
+  result.push(`${label || key}: ${value}`);
+}
 
-function _signingString(
-  {includeHeaders, headers, method, url, host, path, validate}) {
+function _signingString({includeHeaders, headers, method, url,
```

Comment:
Fix unusual line breaking

Reply:
tried to fix this, options are

```js
function _signingString({includeHeaders, headers, method, url,
  host, path, requestOptions, validate}) 

function _signingString(
  {includeHeaders, headers, method, url, host, path, requestOptions, validate}) // to long

function _signingString({
  includeHeaders, headers, method,
  url, host, path, requestOptions, validate})
```
ideas?

Reply:
Last one is acceptable.

Reply:
can we add that suggest to the style guide?

Reply:
> can we add that suggest to the style guide?

Sure, I thought the linter would have already caught it, but maybe something isn't hooked up or the rule isn't covered somehow.

Reply:
Addressed in https://github.com/digitalbazaar/http-signature-header/pull/12/commits/1f18f7e132e46beba7bcca9f7eeca115e1bb90d4


P.S. I believe there was some disagreement on the function parameter length rule so in the end we turned it off. I do agree I preferred having it on.